src/mod_vcard.erl

```diff
@@ -196,6 +204,114 @@ stop(VHost) ->
     gen_server:call(Proc, stop),
     ejabberd_sup:stop_child(Proc).
 
+%%--------------------------------------------------------------------
+%% config_spec
+%%--------------------------------------------------------------------
+
+-spec config_spec() -> mongoose_config_spec:config_section().
+config_spec() ->
+    #section{
+       items = #{<<"iqdisc">> => mongoose_config_spec:iqdisc(),
+                 <<"host">> => #option{type = string,
+                                       validate = non_empty},
+                 <<"search">> => #option{type = boolean},
+                 <<"backend">> => #option{type = atom,
+                                          validate = {module, mod_vcard}},
+                 <<"matches">> => #option{type = int_or_infinity,
+                                          validate = non_negative},
+                 <<"ldap_pool_tag">> => #option{type = atom,
+                                                validate = non_empty},
+                 <<"ldap_base">> => #option{type = string},
+                 <<"ldap_uids">> => #list{items = ldap_uids_spec()},
```

Comment:
This seems identical to `mongoose_config_spec:ldap_uids/0`, maybe reuse that?

Reply:
good point